types/types.go

```diff
@@ -331,6 +331,14 @@ type ImageDestination interface {
 	Commit(ctx context.Context, unparsedToplevel UnparsedImage) error
 }
 
+type ImageDestinationReusableWithRef interface {
```

Comment:
Probably should be `WithSrcRef`, or at the very least clearly documented.

Reply:
Fixed it.